### PR TITLE
Archive Experiment Update (Release TBD)

### PR DESCRIPTION
**Pull request by jimmybrannonLD**
- Added a blurb about archiving experiments with screenshot.
- Updated existing screenshot of experiment header to include the new layout

If ya'll need want more context on the feature I'm happy to give you a demo. You can also find the product and tech spec [here](https://launchdarkly.atlassian.net/wiki/spaces/ENG/pages/2163015726/Archiving+Experiments)

### CODE REVIEW

File: src/content/topics/home/experimentation-creating/index.mdx [hunk 1]
```diff
@@ -134,12 +134,20 @@ To start an experiment iteration:
...
 To learn more about starting and stopping experiment iterations, read [Managing experiments](/home/analyzing-experiments/managing).
 
+## Archiving Experiments
+
+You can archive an experiment if you want to retire it from LaunchDarkly.
+
+Archived experiments are hidden from the experiment dashboard by default, however you still cannot delete a flag or metric attached to an archived experiment.
+
+![An experiment with the "Archive experiment" button called out.](experiment-archive-callout.png)
+
```
Comment (mmrj) [inline src/content/topics/home/experimentation-creating/index.mdx:150]:
```suggestion
## Archiving and restoring experiments

You can archive an experiment if you want to retire it from LaunchDarkly. 

Archived experiments are hidden from the experiment dashboard by default. However, you cannot delete a flag or metric attached to an archived experiment. You also cannot start new iterations for archived experiments.

To archive an experiment:

1. Navigate to the **Experiments** list in the environment you want to start an iteration in.
2. Click on the name of the experiment you want to archive. The experiment detail page appears.
3. Click **Archive experiment**.

![An experiment with the "Archive experiment" button called out.](experiment-archive-callout.png)

To restore an experiment, click **Restore experiment** from the experiment details page. 

```

Suggested updates to:
- use sentence case in section title
- include short procedure on archiving steps
- note that you cannot start iterations for archived experiments
- note that you can restore experiments

For restoring experiments, I noticed this in the tech spec and played with it in staging. If it won't be released at the same time, let's remove that line for now.

**Review by mmrj — CHANGES_REQUESTED** [review]
Thanks for getting this started! Some suggested updates to the new section.

File: src/content/topics/home/experimentation-creating/index.mdx (hunk 1 above)
Comment (jimmybrannonLD) [reply src/content/topics/home/experimentation-creating/index.mdx:150]: oh nice, thanks @mmrj! Restoring _will_ be released at the same time, so we can leave that in :)

File: src/content/topics/home/experimentation-creating/index.mdx [hunk 2]
```diff
@@ -134,12 +134,27 @@ To start an experiment iteration:
...
 To learn more about starting and stopping experiment iterations, read [Managing experiments](/home/analyzing-experiments/managing).
 
+## Archiving and restoring experiments
+
+You can archive an experiment if you want to retire it from LaunchDarkly.
+
+Archived experiments are hidden from the experiment dashboard by default. However, you cannot delete a flag or metric attached to an archived experiment. You also cannot start new iterations for archived experiments.
+
+To archive an experiment:
+1. Navigate to the **Experiments** list in the environment you want to start an iteration in.
```
[the remark this answers is not in the document]
Comment (mmrj) [reply src/content/topics/home/experimentation-creating/index.mdx:150]: My fault, thanks for catching! Updated.

**Review by paigegarnick — APPROVED** [review]

**Pull request merged**